app/services/process_output_documents.rb

```diff
@@ -1,6 +1,15 @@
 class ProcessOutputDocuments
   def call
-    batch = CreateBatch.new.call
+    tags = []
+    tags << 'ProcessOutputDocuments'
+    tags << SecureRandom.uuid
```

Comment:
What's the intent behind `SecureRandom.uuid`? If it is to signify a unique invocation of this method?